src/core/trulens/core/schema/record.py

```diff
@@ -163,7 +163,7 @@ class Record(serial.SerialModel, Hashable):
     def __init__(
         self, record_id: Optional[mod_types_schema.RecordID] = None, **kwargs
     ):
-        super().__init__(record_id="temporary", **kwargs)
+        super(serial.SerialModel).__init__(record_id="temporary", **kwargs)
```

Comment:
Was this a linter suggestion because this class inherits from more than one parent?

Reply:
yea